deploy/yamls/ubuntu/zfs-driver.yaml

```diff
@@ -841,4 +933,225 @@ spec:
+              mountPath: /home/keys
+            - name: zfs-bin
+              mountPath: /sbin/zfs
+            - name: libzpool
+              mountPath: /lib/libzpool.so.2
+            - name: libzfscore
+              mountPath: /lib/libzfs_core.so.1
+            - name: libzfs
+              mountPath: /lib/libzfs.so.2
+            - name: libuutil
+              mountPath: /lib/libuutil.so.1
+            - name: libnvpair
+              mountPath: /lib/libnvpair.so.1
+            - name: exportfs
+              mountPath: /usr/sbin/exportfs
+            - name: mount-nfs
+              mountPath: /usr/sbin/mount.nfs
+            - name: libtirp
+              mountPath: /usr/lib/x86_64-linux-gnu/libtirpc.so.3
+            - name: pods-mount-dir
+              mountPath: /var/lib/kubelet/
+              mountPropagation: "Bidirectional"
+              # needed so that any mounts setup inside this container are
+              # propagated back to the host machine.
+            - name: libgssapi
+              mountPath: /usr/lib/x86_64-linux-gnu/libgssapi_krb5.so.2
+            - name: libkrb
+              mountPath: /usr/lib/x86_64-linux-gnu/libkrb5.so.3
+            - name: libkrbcrypto
+              mountPath: /usr/lib/x86_64-linux-gnu/libk5crypto.so.3
+            - name: libkrbsupport
+              mountPath: /usr/lib/x86_64-linux-gnu/libkrb5support.so.0
+            - name: libkeyutils
+              mountPath: /usr/lib/x86_64-linux-gnu/libkeyutils.so.1
+            - name: exports
+              mountPath: /etc/exports
+            - name: nfs-state
+              mountPath: /var/lib/nfs
+      volumes:
+        - name: device-dir
+          hostPath:
+            path: /dev
+            type: Directory
+        - name: encr-keys
+          hostPath:
+            path: /home/keys
+            type: DirectoryOrCreate
+        - name: zfs-bin
+          hostPath:
+            path: /sbin/zfs
+            type: File
+        - name: libzpool
+          hostPath:
+            path: /lib/libzpool.so.2.0.0
+            type: File
+        - name: libzfscore
+          hostPath:
+            path: /lib/libzfs_core.so.1.0.0
+            type: File
+        - name: libzfs
+          hostPath:
+            path: /lib/libzfs.so.2.0.0
+            type: File
+        - name: libuutil
+          hostPath:
+            path: /lib/libuutil.so.1.0.1
+            type: File
+        - name: libnvpair
+          hostPath:
+            path: /lib/libnvpair.so.1.0.1
+            type: File
+        - name: exportfs
+          hostPath:
+            path: /usr/sbin/exportfs
+            type: File
+        - name: mount-nfs
+          hostPath:
+            path: /usr/sbin/mount.nfs
+            type: File
+        - name: libtirp
+          hostPath:
+            path: /usr/lib/x86_64-linux-gnu/libtirpc.so.3
+            type: File
+        - name: libgssapi
+          hostPath:
+            path: /usr/lib/x86_64-linux-gnu/libgssapi_krb5.so.2
```

Comment:
question: which os you are using and which zfs version? I have seen this library is needed for centos

Reply:
I am on Ubuntu Focal using zfs 0.8.3-1ubuntu12.1.